psycop/common/model_training_v2/trainer/data/test_dataloaders.py

```diff
@@ -33,3 +38,18 @@ def test_vertical_concatenator(tmpdir: Path):
         ParquetVerticalConcatenator(
             paths=[str(p) for p in parquet_paths] + ["non_existent_path"],
         ).load()
+
+
+def test_dataloader_filterer():
+    base_dataloader = MinimalTestData()
+
+    class TestDataFilter:
```

Comment:
```suggestion
    class MockDataFilter:
```

A naming convention I've come across! Helps pytest not hit the "test elements" if they are top-level (e.g. a func), and is consistent with other languages' terminology 👍